gdal/frmts/hdf5/bagdataset.cpp

```diff
@@ -4582,8 +4569,6 @@ void GDALRegister_BAG()
 "  <Option name='ZLEVEL' type='int' "
     "description='DEFLATE compression level 1-9' default='6' />"
 "  <Option name='BLOCK_SIZE' type='int' description='Chunk size' />"
-"  <Option name='CORNER_POINTS_EXTEND_HALF_PIXEL' type='boolean' description="
```

Comment:
I'm OK if we drop that capability, but then it must be undocumented as well in doc/source/drivers/raster/bag.rst